docs/_pages/releases.md

```diff
@@ -20,6 +20,7 @@ sidebar:
 * Changed AttributeBasedFormatter to allow custom formatter selection based on the parent type - [#1418](https://github.com/fluentassertions/fluentassertions/pull/1418).
 * Added nullable overload for `Be` and `NotBe` methods of `DateTimeAssertions` and `DateTimeOffsetAssertions` - [#1427](https://github.com/fluentassertions/fluentassertions/issues/1427).
 * Added overload of `Enumerating` extension method to be able to force the enumeration of an object member -[#1433](https://github.com/fluentassertions/fluentassertions/pull/1433)
+* add overloads of MatchRegex and NotMatchRegex that take System.Text.RegularExpressions.Regex -[#1436](https://github.com/fluentassertions/fluentassertions/pull/1436)
```

Comment:
🙃 For consistency, can you please start sentences with a capital and wrap code elements with back-ticks?